app/views/files/index.html.erb

```diff
@@ -16,7 +16,9 @@
           </section>
 
           <section class="file-preview">
-            <% if attachment.image? %>
+            <% if present(attachment).thumbnail(:thumb).present? %>
+              <div class="image-preview" style="background-image: url(<%= present(attachment).thumbnail(:thumb) %>);"></div>
```

Comment:
probably not great to be call the `present(...)` twice here, but i imagine we'll need to refactor a lot of this down the line when we start pulling content out of work docs etc?